api/handlers/userhandler.py

```diff
@@ -23,7 +23,12 @@ def __init__(self, request=None, response=None):
     def get(self, _id):
         user = self._get_user(_id)
         permchecker = userauth.default(self, user)
-        result = permchecker(self.storage.exec_op)('GET', _id, projection={'api_key': 0} or None)
+        projection = {'api_key': 0}
+        if self.user_is_admin:
+            projection['wechat.openid'] = 0
+        else:
+            projection['wechat'] = 0
```

Comment:
Dumb question: will this cause `wechat` objects to be returned for users that don't have a `wechat` field?